test/catch2/live_restore/api/test_live_restore_fs_open_file.cpp

```diff
@@ -106,36 +106,38 @@ TEST_CASE("Live Restore fs_open_file", "[live_restore],[live_restore_open_file]"
 
     SECTION("fs_open - Directory")
     {
+        /*
+         * WiredTiger never creates directories. The user must do this outside of WiredTiger. This
+         * means all cases where the directory doesn't exist in the destination we return ENOENT
+         * These cases include when:
```

Comment:
I see what you're doing here but it seemed like the comment ended as "incomplete". Maybe move the declaration of `lr_fh` above the comment?